### DISCUSSION TITLE
(O)DoH over HTTP

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2480] [Q&A]
What's [(O)DoH over HTTP](https://github.com/DNSCrypt/dnscrypt-proxy/commit/8bea679e7b97e152620ae90033b2f193694d4f61) and which servers support it?
As far as I understand it's _un_encrypted DNS, i.e. neither DoH nor ODoH.

**Comment by ghost** [comment]
I'm going to report a CVE for this. Any comments?

**Comment by jedisct1** [comment]
There's no way to use HTTP, it's not even supported by DNS stamps.

What that code does is avoid an empty pointer deference when printing log messages if there's no TLS information. It's only for debugging.
